<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPage</class>
 <widget class="QDialog" name="UserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(104, 161, 229);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>901</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border-radius: 3px; 
	background-color: rgb(242, 248, 253);
 } </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string> Report Issue</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>261</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Report any issues you may &lt;/p&gt;&lt;p&gt;encounter. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelIconSend2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReportIssue">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
       </property>
       <property name="text">
        <string>Detail</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border-radius: 3px; 
	background-color: rgb(242, 248, 253);
 } </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>View Vaccination Record</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>View your Vaccination records.</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelIconView2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonVaccinationRecord">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
       </property>
       <property name="text">
        <string>Detail</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border-radius: 3px; 
	background-color: rgb(242, 248, 253);
 } </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Request Vaccine Pass</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Request your Vaccine Pass.</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelIconPass">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonVaccinePass">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
       </property>
       <property name="text">
        <string>Detail</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border-radius: 3px; 
	background-color: rgb(242, 248, 253);
 } </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>View RAT Result</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>281</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>View your Covid-19 test results.</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelIconView1">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCovid19TestResult">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
       </property>
       <property name="text">
        <string>Detail</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border-radius: 3px; 
	background-color: rgb(242, 248, 253);
 } </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Report RAT Result</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>.AppleSystemUIFont</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Report your RAT Results.</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelIconSend1">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>0</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(242, 248, 253,0)</string>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRATResult">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
       </property>
       <property name="text">
        <string>Detail</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>75</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(104, 161, 229);</string>
      </property>
      <property name="text">
       <string>My Health Account </string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>368</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="styleSheet">
       <string notr="true">QGroupBox { 
     border-radius: 3px; 
	background-color: rgb(242, 248, 253);
 } </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelIconUser">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(242, 248, 253,0)</string>
         </property>
         <property name="text">
          <string>Icon</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelUserName">
           <property name="font">
            <font>
             <family>.AppleSystemUIFont</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(242, 248, 253,0)</string>
           </property>
           <property name="text">
            <string>User Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonViewProfile">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0,0.1); 
	border: none; 
	color: rgb(0, 0, 0); 
	font: 12px;
} 

QPushButton:hover { 
	background-color: rgb(0, 0, 0, 0.1); 
	border: none; 
	color: rgb(0, 43, 255); 
}</string>
           </property>
           <property name="text">
            <string>View Profile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonLogout">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: rgb(242, 189, 76); 
    border: none;
     border-radius: 5px; 
    color: rgb(0, 0, 0); 
    font: 16px;
	padding: 5 20px;
} 

QPushButton:hover { 
    background-color: rgb(245, 149, 66); 
    border: none; 
    border-radius: 5px; 
    color: rgb(0, 43, 255); 
}</string>
        </property>
        <property name="text">
         <string>Log out</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionview">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/view.png</normaloff>:/Images/view.png</iconset>
   </property>
   <property name="text">
    <string>view</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
